src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net48</TargetFrameworks>
```

Comment:
>net48 [](http://example.com/codeflow?start=76&length=5)

if we bump to net48, you won't be able to use sarifconverters from a netframework 462, for example.

in this case, my suggestion is to change to 462 (the lowest supported version).

Reply:
Yes, this was a mistake. What I'm going to do is drop this to net45. As mentioned previously, although net462 is the lowest supported version, it is not appropriate for our dependency to require users to upgrade to the supported version, that is their business decision to make.